Aptoide/aptoide-client-v8
# [ASV-2166] Implemented fiat values from server in promotions fragment;

**What does this PR do?**

This PR implements messages and fiat values for promotions coming from server; It also changes the look and feel of the cards on the fragment, as per zeplin;

**Database changed?**

No

**Where should the reviewer start?**

- [ ] PromotionsFragment.java

**How should this be manually tested?**

Use the rewrites in the ticket (this is not in production as of yet) and make sure everything is according to zeplin and that the messages shown match what you're getting from charles;

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-2166](https://aptoide.atlassian.net/browse/ASV-2166)

**Code Review Checklist**
